runbotics/runbotics-scheduler/src/websocket/bot/process-launch/bot-process-instance.service.ts

```diff
@@ -85,12 +87,33 @@ export class BotProcessService {
     async updateProcessLastRunTime(processInstance: IProcessInstance) {
         if (!processInstance?.process?.id) return;
         const process = await this.processService.findById(processInstance.process.id);
-        
+
         if (process && processInstance.created) {
             await this.processService.partialUpdate({ id: process.id, lastRun: processInstance.created });
         }
     }
 
+    async notifyAboutProcessInstanceStatus(processInstance: IProcessInstance) {
+        const statusList = [
+            ProcessInstanceStatus.INITIALIZING,
```

Comment:
initializing happens at the beginning of the process even before `in_progress` so we don't want to inform the user about it, only when completed or error